spec/rails_helper.rb

```diff
@@ -34,6 +34,9 @@
 Capybara.javascript_driver = :poltergeist_silent # uncomment this to disable console.log (including warn)
 Capybara.default_max_wait_time = 3
 
+# Fix load order issue on Circle CI 2.0
+require File.expand_path('../support/pages/base.rb', __FILE__)
+
```

Comment:
As the comment said, running tests on CircleCI failed because of load order differences as dependencies are included on the different architecture on CircleCI 2.0 compared to local machine